### ISSUE TITLE
[ApiView] Don't select line numbers when copying

### ISSUE THREAD

**Issue by tg-msft** [Azure/azure-sdk-tools#4198]
Sometimes I need to copy the API listing to diff manually ([example](https://gist.github.com/tg-msft/647eaf5f83399c9b0a0f3451da2b2f50/revisions)), but the line numbers make a mess of it.  Can we add `user-select: none;` to [the `.line-number` class](https://github.com/Azure/azure-sdk-tools/blob/e8486c4bd14259034718d945a9d6bb32c9e4f3e7/src/dotnet/APIView/APIViewWeb/Client/css/site.scss#L878-L884)?

**Comment by praveenkuttappan** [comment]
@tg-msft Line numbers are not copied if you select hide line number option. Will this be good enough?

![image](https://user-images.githubusercontent.com/55455725/200626204-dcc09241-fc58-48b1-a1a5-abb3ee794b21.png)

I also would like to know the use case when you need to diff manually. Please let me know if there is any improvement, we can do in APIView to cover any missing use cases.

**Comment by tjprescott** [comment]
Yes, because SwiftAPIView can only do manual reviews, you sometimes get a diff due to changes in SwiftAPIView itself, which is I suspect the scenario where @tg-msft is doing manual diffs. We've written guidance for SwiftAPIView [here](https://github.com/Azure/azure-sdk-for-ios/wiki/Using-Swift-APIView) so that service teams _should_ regenerate the base revisions so that all diffs are done against the same version.

However, I would agree with Ted that no one would ever _want_ to copy the line numbers from APIView, so it would be wonderful if it just didn't copy them (with or without "show line numbers").
